index.html

```diff
@@ -24,17 +24,127 @@ <h1 class="main_content">
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="button-flex">See Project</button>
+                </div>
+                <div class="grid-item">
+                    <div class="img-holder"></div>
+                    <h3 class="text3">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="cate">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="button-flex">See Project</button>
+                </div>
+             </div>
+            </section>
+            <section class="about-me-section">
+                    
+                <div class="about-me">
+                    <img class="il-about il-1-a" src="illustration 1- about me-mobile.svg">
+                     <img class="il-about il-2-a" src="illustration 2-about me mobile.svg">
+                    <h2 class="h2-about">About me</h2>
+                    <p class="p-about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.</p>
+                     <button class="button-about">Get My Resume</button>
+             </section>
+                    </div>
+                <div class="about-me-grid">
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/icon-languages.svg"></div>
+                            <h3 class="text3-about">Languages</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/Icons.svg"></div>
+                            <h3 class="text3-about">Frameworks</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/ligth.svg"> </div>
+                            <h3 class="text3-about">Skills</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                </div>
```

Comment:
- [ ] Kindly make sure to wrap all these two containers in one section as all hold the information about one section which is `About Myself`. It's better and required all of them in one `section` conatiner.